src/modules/ekf2/module.yaml

```
@@ -24,6 +24,18 @@ parameters:
       default: 7
       min: 0
       max: 7
+    EKF2_DELAY_MAX:
+      description:
+        short: Maximum delay of all the aiding sensors
+        long: Defines the delay between the current time and the delayed-time horizon.
+          This value should be at least as large as the largest EKF2_XXX_DELAY parameter.
+      type: float
+      default: 110
+      min: 0
+      max: 300
```

I've seen at least 800 ms. 

```suggestion
      max: 1000
```